index.html

```diff
@@ -611,10 +738,10 @@ <h4>Script Type</h4>
                   and MAY also contain <a>Original</a> <a>Text</a> objects from the <a>Original Language Transcript</a>
                   or <a>Translation</a> <a>Text</a> objects in other languages for context and quality verification.
                   They MAY also contain links to audio and mixing instructions for the purpose of producing an audio track incorporating the recordings.</p>
-                <aside class="note">Even though the output of the dubbing workflow is in the same language as those
-                <a>Text</a> objects with the matching language,
-                the <a>Text Language Source</a> of those Text objects remains <a>Translation</a>
-                since they differ from the original language of the dialogue.</aside>
+                <aside class="note"><a>As-recorded Scripts</a> are not considered to be transcripts
+                  even though they are adjusted to match the recorded output of the dubbing workflow.
```

Comment:
I propose to delete the addition  "As-recorded Scripts</a> are not considered to be transcripts even though they are adjusted to match the recorded output of the dubbing workflow." It would make the note clearer if it just contains the second sentennce.

Reply:
Done in 10b1a0e